examples/ui.md

```diff
@@ -491,6 +491,21 @@ You can also get an alert with more padding and large icon by specifying `large`
 <x-icon name="chevron-down" size="xs" class="md:h-3 md:w-2" />
 ```
 
+### Chevron Down/Up component
+
+Shows the chevron icon which rotates based on specific criteria
+
+```blade
+<div x-data="{ containerOpen: false }">
+    <x-ark-chevron-open-close is-open="containerOpen === true" />
+</button>
```

Comment:
The example seems not in line with the component...

```suggestion
<x-ark-chevron-open-close is-open="{{ true }}" size="sm" />
```

Reply:
The example is intentionally like this :thinking: the `is-open` should be plaintext, and not php since it's a js/alpine condition. In this case, it will show it as "open" if `containerOpen` is true from an alpine action. I have added the size though

Reply:
@alexbarnsley 🤔 you're right!